src/core/lifecycle/create-props.js

```diff
@@ -62,6 +62,12 @@ function getDefaultProps(layerClass) {
   }
 
   const parent = layerClass.prototype;
+  if (!parent) {
+    return {
```

Comment:
`return null` should work

Reply:
> return null should work

I think it would break on line 11.

Reply:
Assuming `createProps` is always called by the `Layer` class, line 11 will not get `null`.

Reply:
You are probably right. I'd prefer to let it stand as is, just to minimize churn. The reason is that the code is safe/working as it is, and I just confirmed that this line will be changed in successive, staged diffs anyway. 

My assumption is that the main reason you requested changes was the issue with the `updateTriggers` forwarding, which I addressed, not this point, right?